cmd/main.go

```diff
@@ -55,7 +56,19 @@ func newClient(canaryConfig *config.CanaryConfig) sarama.Client {
 	config.Producer.Partitioner = sarama.NewManualPartitioner
 	config.Producer.Return.Successes = true
 	config.Producer.RequiredAcks = sarama.WaitForAll
-	// TODO: handling error
-	client, _ := sarama.NewClient([]string{canaryConfig.BootstrapServers}, config)
-	return client
+
+	backoff := services.NewBackoff(canaryConfig.BootstrapMaxAttempts, 5000)
```

Comment:
Would it make sense to have the `scale` be configurable as well as the attempts? If you set attempts to a large number your backoff times could get huge?

Reply:
Makes sense. Just did the change.